app/controlplane/pkg/metrics/prometheus/registry/registry.go

```diff
@@ -28,19 +28,34 @@ type PrometheusRegistry struct {
 	*prometheus.Registry
 	Name               string
 	chainloopCollector *collector.ChainloopCollector
+
+	// metrics
+	WorkflowRunDurationSeconds *prometheus.HistogramVec
 }
 
+var workflowRunDurationSeconds = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+	Name: "chainloop_wf_run_duration_seconds",
```

Comment:
In the case of the already existing metric, it's called: `chainloop_workflow_up`, in order to be consistent with the naming, shall we name this `chainloop_workflow_run_duration_seconds` ?

I don't think we should be mixing short names with extended ones.

Reply:
Done, ptal